spec/services/staff_action_logger_spec.rb

```diff
@@ -546,4 +546,63 @@
     end
 
   end
+
+  describe 'log_topic_closed' do
+    fab!(:topic) { Fabricate(:topic) }
+
+    it "raises an error when argument is missing" do
+      expect { logger.log_topic_closed(nil) }.to raise_error(Discourse::InvalidParameters)
+    end
+
+    it "creates a new UserHistory record" do
+      expect { logger.log_topic_closed(topic, closed: true) }.to change { UserHistory.count }.by(1)
+      user_history = UserHistory.last
+      expect(user_history.action).to eq(UserHistory.actions[:topic_closed])
```

Comment:
```suggestion
      expect { logger.log_topic_closed(topic, closed: true) }.to change { UserHistory.where(action: UserHistory.actions[:topic_closed].count) }.by(1)
```

Reply:
Moved the `.count` outside of the parens, but yes: done